code/modules/surgery/advanced/pacification.dm

```diff
@@ -0,0 +1,40 @@
+/obj/item/disk/surgery/pacification
+	desc = "The disk provides instructions on how to suppress violence by manipulating the patient's brain."
+	surgeries = list(/datum/surgery/advanced/pacify)
+
+/datum/surgery/advanced/pacify
+	name = "violence neutralization"
+	steps = list(/datum/surgery_step/incise,
+				/datum/surgery_step/retract_skin,
+				/datum/surgery_step/saw,
+				/datum/surgery_step/clamp_bleeders,
+				/datum/surgery_step/pacify,
+				/datum/surgery_step/close)
+
+	species = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list("head")
+	requires_bodypart_type = 0
+
+/datum/surgery/advanced/pacify/can_start(mob/user, mob/living/carbon/target)
+	. = ..()
+	var/obj/item/organ/brain/B = target.getorganslot(ORGAN_SLOT_BRAIN)
+	if(!B)
+		return FALSE
+
+/datum/surgery_step/pacify
+	name = "rewire brain"
+	implements = list(/obj/item/hemostat = 100, /obj/item/screwdriver = 35, /obj/item/pen = 15)
+	time = 40
+
+/datum/surgery_step/pacify/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] begins to reshape [target]'s brain.", "<span class='notice'>You begin to reshape [target]'s brain...</span>")
+
+/datum/surgery_step/pacify/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] reshapes [target]'s brain!", "<span class='notice'>You succeed in reshaping [target]'s brain.</span>")
+	target.gain_trauma(/datum/brain_trauma/severe/pacifism, TRAUMA_RESILIENCE_LOBOTOMY)
+	return TRUE
+
+/datum/surgery_step/pacify/failure(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] reshapes [target]'s brain!", "<span class='notice'>You screwed up, and rewired [target]'s brain the wrong way around...</span>")
+	target.gain_trauma_type(BRAIN_TRAUMA_SEVERE, TRAUMA_RESILIENCE_LOBOTOMY)
```

Comment:
should throw the patient into a brief unstunnable rage and give them hulk so if you mess up pacifying somebody they can smash down a wall and go on a rampage